### DISCUSSION TITLE
Need a example for function call argument list

### DISCUSSION THREAD

**Discussion by windwhisper4759** [foonathan/lexy#186] [Q&A] [answered]
I need to parse a string like "func(0.1, 0.2, 1.3)"  "func(0.21)".
The number of arguments is arbitrary.
How to do it?

**Comment by foonathan** [comment, marked as answer]
https://lexy.foonathan.net/playground/?id=G4qxvzn8x&mode=tree

You probably also want to integrate that into `dsl::expr`: https://lexy.foonathan.net/reference/dsl/expression/#doc

A function call is a postfix expression using `dsl::op(dsl::parenthesized.opt_list(...))` as operator, and the function name as operand.
